tests/test_leakage.py

```diff
@@ -116,7 +116,9 @@ def test_find_leak_assymetric(self):
         )
 
         # Test looking up inverse triples
-        test_leaked = test_factory.get_triples_for_relations(sealant.inverse_relations_to_delete)
+        test_leaked = test_factory.mapped_triples[
+            test_factory.get_mask_for_relations(relations=sealant.inverse_relations_to_delete, invert=False)
```

Comment:
this is now the only place TriplesFactory.get_mask_for_relations is being used, right? maybe not necessary to test anymore?

Reply:
It is also used in `new_with_restriction`